lib/WWW/Splunk/API.pm

```diff
@@ -176,13 +176,6 @@ sub request {
 	$self->{agent}->add_handler (response_header => sub {
 		my($response, $ua, $h) = @_;
 
-		# Deal with HTTPS errors
-		# newer LWP::UserAgent does this right
-		if ($_ = $response->header ('Client-SSL-Warning')) {
-			# Why does LWP tolerate these by default?
-			croak "SSL Error: $_" unless $self->{unsafe_ssl};
-		}
-
```

Comment:
This Client-SSL-Warning is set by LWP::Protocol::https in situations, which it's a warning but not problem. One is turning of ssl cert verification and second is setting of schema.

And in history of LWP::UserAgent was set incidently
    if(! eval { $sock->get_peer_verify }) {
       $res->header("Client-SSL-Warning" => "Peer certificate not verified");
    }
called in IO::Socket::SSL, which not have get_peer_verify method.